### PR TITLE
bump version pointers in integration test

### PR DESCRIPTION
**Pull request by kvmware**
part of release https://github.com/knative/release#knativeclient

### CODE REVIEW

File: test/presubmit-integration-tests-latest-release.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@
 # This script is used in Knative/test-infra as a custom prow job to run the
 # integration tests against Knative Serving / Eventing of a specific version.
 
-export KNATIVE_SERVING_VERSION="`git ls-remote --heads git@github.com:knative/serving.git | grep release- | cut -d '-' -f2 | sort -r | head -n 1`"
-export KNATIVE_EVENTING_VERSION="`git ls-remote --heads git@github.com:knative/eventing.git | grep release- | cut -d '-' -f2 | sort -r | head -n 1`"
+export KNATIVE_SERVING_VERSION="1.3.0"
+export KNATIVE_EVENTING_VERSION="1.3.0"
```
Comment (rhuss) [inline test/presubmit-integration-tests-latest-release.sh:21]: @kvmware why was this change needed ? // cc @dsimansk

Comment (dsimansk) [reply test/presubmit-integration-tests-latest-release.sh:21]: With this change being merged first, the version bump in e2e isn't needed anymore. At least that was the intention, right? https://github.com/knative/client/pull/1623

However, I've looked closer at #1623 and I don't think it's grabbing the correct version. In the log of "integration-tests" the URLs point to nigthly yamls, instead of versioned ones. I'll add comment directly to the PR.

https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative_client/1623/pull-knative-client-integration-tests-latest-release/1501297657442734080#1:build-log.txt%3A361

**Comment by kvmware** [comment]
https://github.com/knative/client/pull/1625 should fix this
